jhove-core/src/main/java/edu/harvard/hul/ois/jhove/NisoImageMetadata.java

```diff
@@ -20,6 +20,11 @@
 	/** Logger for this class. */
     private static final Logger LOGGER = Logger.getLogger(NisoImageMetadata.class.getCanonicalName());
 
+    /** private String constants. */
+    private static final String YES = "Yes";
+    private static final String NO = "No";
+    private static final String TIME_STEP = ":";

```

Comment:
I hate to do this but I meant `TIME_SEP` sorry, short for time separator as that's effectively its use here, sorry I know I'm fussy, it's a general developer character failing and job requirement. :)